GSA/Challenge_platform
# [181] Evaluation Form Validation - Part 1

#181 

## Reason for changes
Persisted checks were originally added so new forms didn't show criteria sections like option labels before a radio button was checked and to make sure there were no default selected options. 

I switched this to just checking if the value exists since a form validation error would cause parts of the form to not display on newly created forms if they failed validation since they technically didn't "persist"

## Steps to test
- Create a new eval form
- Fill out all form data
- Add at least one criteria with either binary/rating scale to see option labels
- Force a validation error that still allows submission in one of two ways:
  - Have two criteria with the same title
  - Have scale type as weighted with criteria that do not sum to 100
- Form should still have the scale type selected
- Form should still show option labels for binary/rating criteria

Same should hold true on editing a form
